datum/umbrel-app.yml

```
@@ -0,0 +1,26 @@
+manifestVersion: 1.1
+id: datum
+category: Bitcoin
+name: Datum
+version: "1"
+tagline: Make your own mining template
+icon: https://raw.githubusercontent.com/Retropex/Bitcoin-store/refs/heads/datum/btc-datum/icon/datum_logo.svg
+description: >-
+  Take control of your digital sovereignty by running your own datum server.
+
+
+  Powered by Bitcoin Knots: https://bitcoinknots.org/
+developer: Ocean
+website: https://ocean.xyz
+dependencies:
+  - bitcoin-knots
+repo: https://github.com/ocean-xyz
+support: https://ocean.xyz
+port: 21000
+gallery:
+  - https://raw.githubusercontent.com/Retropex/Bitcoin-store/refs/heads/datum/btc-datum/icon/preview.png
+path: ""
+defaultPassword: ""
+releaseNotes: >-
+submitter: Léo Haf
+submission: https://github.com/Retropex/Bitcoin-store/pull/1
```

We can change this now to this submission:
```suggestion
submission: https://github.com/getumbrel/umbrel-apps/pull/1661
```